sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -36,6 +36,8 @@ private[spark] object SQLConf {
   val PARQUET_INT96_AS_TIMESTAMP = "spark.sql.parquet.int96AsTimestamp"
   val PARQUET_CACHE_METADATA = "spark.sql.parquet.cacheMetadata"
   val PARQUET_COMPRESSION = "spark.sql.parquet.compression.codec"
+  val PARQUET_BLOCK_SIZE = "spark.sql.parquet.blocksize"
+  val PARQUET_PAGE_SIZE = "spark.sql.parquet.pagesize"
```

Comment:
I meant Parquet should have its own conf keys for those, right? I do not need to add spark sql ones.

See https://github.com/apache/parquet-mr/blob/parquet-1.6.0rc3/parquet-hadoop/src/main/java/parquet/hadoop/ParquetOutputFormat.java#L98-L106